eclipse-scout-core/src/form/fields/sequencebox/SequenceBox.js

```diff
@@ -183,6 +185,7 @@ export default class SequenceBox extends CompositeField {
 
     // Make sure the last field won't display a status (but shows status CSS class)
     this._lastVisibleField.setSuppressStatus(FormField.SuppressStatus.ICON);
+    this._lastVisibleField.on('propertyChange:suppressStatus', this._lastVisibleFieldSuppressStatusHandler);
```

Comment:
The listeners needs to be removed when the sequencebox is removed, otherwise it will create a memory leak.

I just noticed that there are already some listeners creating a leak:
- propertyChange listener added at _render
- destroy listener added at setParent of Widget caused by FieldStatus that is only removed and not destroyed

Can you please fix the propertyChange listener as well? I will have a look at the FieldStatus.
Instead of adding a custom suppressStatus listener, you could add your code to the existing propertyChange listener, what do you think?